rasa/core/policies/ted_policy.py

```diff
@@ -1173,6 +1217,18 @@ def __init__(
             entity_tag_specs: the entity tag specifications
         """
         super().__init__("TED", config, data_signature, label_data)
+        try:
+            # Disable all GPUS
+            tf.config.set_visible_devices([], 'GPU')
```

Comment:
Why is this needed if you have done the correct device placement in `run_training`?